src/components/Reports/StaffExpenseReport/StaffExpenseReport.tsx

```diff
@@ -0,0 +1,587 @@
+import { PrintTables } from './Tables/PrintTables';
+import { StaffReportTable } from './Tables/StaffReportTable';
+import { filterTransactions } from './filterTransactions';
+
+const ScreenOnly = styled(Box)({
+  '@media print': {
+    display: 'none',
+  },
+});
+
+const PrintOnly = styled(Box)({
+  display: 'none',
+  '@media print': {
+    display: 'block',
+  },
+});
+
+const StyledHeaderBox = styled(Box)({
+  display: 'flex',
+  alignItems: 'center',
+  gap: theme.spacing(2),
+  justifyContent: 'space-between',
+});
+
+const StyledPrintButton = styled(Button)({
+  border: '1px solid',
+  borderRadius: theme.spacing(1),
+  marginLeft: theme.spacing(2),
+  paddingLeft: theme.spacing(2),
+  paddingRight: theme.spacing(2),
+  paddingTop: theme.spacing(1),
+  paddingBottom: theme.spacing(1),
+});
+
+const StyledTimeNavBox = styled(Box)({
+  display: 'flex',
+  margin: 0,
+  gap: theme.spacing(2),
+});
+
+const StyledFilterButton = styled(Button)({
+  color: theme.palette.cruGrayDark.main,
+  borderColor: theme.palette.cruGrayDark.main,
+  '&:hover': {
+    backgroundColor: theme.palette.cruGrayLight.main,
+    borderColor: theme.palette.cruGrayDark.main,
+  },
+});
+
+export interface Transaction extends BreakdownByMonth {
+  fundType: string;
+  category: TransactionCategory['category'];
+  subcategory?: SubCategory['subCategory'];
+}
+
+interface StaffExpenseReportProps {
+  accountId: string;
+  isNavListOpen: boolean;
+  onNavListToggle: () => void;
+  title: string;
+  time: DateTime;
+  setTime: (time: DateTime) => void;
+}
+
+export const StaffExpenseReport: React.FC<StaffExpenseReportProps> = ({
+  isNavListOpen,
+  onNavListToggle,
+  title,
+  time,
+  setTime,
+}) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const { enqueueSnackbar } = useSnackbar();
+  const [isSettingsOpen, setIsSettingsOpen] = useState(false);
+  const [filters, setFilters] = useState<Filters | null | undefined>(null);
+  const [isFilterDateSelected, setIsFilterDateSelected] = useState(
+    Boolean(
+      filters &&
+        (filters.startDate || filters.endDate || filters.selectedDateRange),
+    ),
+  );
+
+  const { data, loading } = useReportsStaffExpensesQuery({
+    variables: {
+      accountId: '1000000001',
+      startMonth:
+        filters?.startDate?.startOf('month').toISODate() ??
+        filters?.endDate?.startOf('month').toISODate() ??
+        time.startOf('month').toISODate(),
+      endMonth:
+        filters?.endDate?.endOf('month').toISODate() ??
+        time.endOf('month').toISODate(),
+    },
+  });
+  const handlePrint = () => window.print();
+
+  const timeTitle = time.toJSDate().toLocaleDateString(locale, {
+    month: 'long',
+    year: 'numeric',
+  });
+
+  const hasNext = time.hasSame(DateTime.now(), 'month');
+
+  const allFunds: Fund[] = data?.reportsStaffExpenses?.funds ?? [];
+
+  const defaultFundType: string | null =
+    allFunds.find((f) => f.fundType === 'Primary')?.fundType ??
```

Comment:
We try to avoid one-letter variable names for code clarity.

```suggestion
    allFunds.find((fund) => fund.fundType === 'Primary')?.fundType ??
```